Doc Fixes (#2495)
python-telegram-bot/python-telegram-bot: examples/README.md

All examples are licensed under the [CC0 License](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/LICENSE.txt) and are therefore fully dedicated to the public domain. You can use them as the base for your own bots without worrying about copyrights.

> Do note that we ignore one pythonic convention. Best practice would dictate, in many handler callbacks function signatures, to replace context with an underscore, since context is an unused local variable in those callbacks. However, since these are examples and not having context present lead to confused beginners, we decided to have it present.

Author: ```suggestion
Do note that we ignore one pythonic convention. Best practice would dictate, in many handler callbacks function signatures, to replace the argument `context` with an underscore, since `context` is an unused local variable in those callbacks. However, since these are examples and not having a name for that argument confuses beginners, we decided to have it present.
```
